common/azure-common/src/main/java/com/azure/common/exception/AzureLibraryException.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.azure.common.exception;
+
+/**
+ * This is internal base exception for Azure Library .
+ */
+public class AzureLibraryException extends AzureException {
+
+    /**
+     * Initializes a new instance of the AzureLibraryException class.
+     *
+     * @param message the exception message.
+     */
+    public AzureLibraryException(String message) {
+        super(message);
+    }
+
+    /**
+     * Initializes a new instance of the AzureLibraryException class.
+     *
+     * @param message the exception message.
+     * @param cause   the Throwable which caused the creation of this AzureException.
```

Comment:
Don't align columns of text like this.

Reply:
The constructor is for the basic use. Do you think it is unnecessary to have the Java doc for the subclass constructor?

Reply:
We should have JavaDoc everywhere we have public API.

Reply:
My point I'm making is don't line up the two 'the' words - just have them appear directly after the param name.

Reply:
Got you. Thanks for pointing it out.